RepairsListener/SqsFunction.cs

```diff
@@ -85,17 +85,24 @@ private async Task ProcessMessageAsync(SQSEvent.SQSMessage message, ILambdaConte
                     {
                         case EventTypes.AssetCreatedEvent:
                             {
-                                processor = ServiceProvider.GetService<ICreateAssetUseCase>();
+                                LambdaLogger.Log("Received a valid Asset Created event. Not doing anything with this for now");
```

Comment:
Add some easy-to-get individual ID such as the message or correlation ID, otherwise the message may be lost in sampling

Reply:
Done